flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/SAI/A320_Neo_SAI.js

```diff
@@ -1239,15 +1239,15 @@ class A320_Neo_SAI_SelfTest extends NavSystemElement {
         this.getDeltaTime = A32NX_Util.createDeltaTimeCalculator();
         const interval = 5;
         this.getFrameCounter = A32NX_Util.createFrameCounter(interval);
-        const cold_dark = SimVar.GetSimVarValue('L:A32NX_COLD_AND_DARK_SPAWN', 'Bool');
+        const coldDark = Boolean(SimVar.GetSimVarValue('L:A32NX_COLD_AND_DARK_SPAWN') && SimVar.GetSimVarValue('L:A32NX_FMGC_FLIGHT_PHASE', 'Enum') === (FmgcFlightPhases.PREFLIGHT));
```

Comment:
**thought**: The `Boolean()` invocation is unnecessary.
**suggestion**: Remove it.

Reply:
Done